### PR TITLE
Rebuild for Python 3.10

### PR DESCRIPTION
**Pull request by raimis**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by raimis** [comment]
@conda-forge-admin, please rerender

**Comment by raimis** [comment]
@peastman I need the OpenMM-PLUMED package for Python 3.10 and Linux.

Some OSX builds are failing because the corresponding OpenMM packages (https://anaconda.org/conda-forge/openmm/files?version=7.7.0) are broken. I assume that we can ignore this for OpenMM 7.7 and it will be fixed for OpenMM 8.0.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,14 +30,14 @@ requirements:
     - python
     - pip
     - openmm
-    - plumed >=2.7
```
Comment (peastman) [inline recipe/meta.yaml:33]: Is this valid syntax?  If you're trying to pin to a particular version, shouldn't it be `==2.7`?

Comment (raimis) [reply recipe/meta.yaml:33]: Yes, it is correct. The tests run with the expected version.

**Review by peastman — APPROVED** [review]
Looks good.

**Pull request merged**